crAPI/Dockerfile

```diff
@@ -0,0 +1,167 @@
+# Build image for web
```

Comment:
I think it is better that this deploy be in a subdir, just like we have K8s, Vagrant in subdirs.

Otherwise, it is not clear that this Dockerfile is for a single container image. The quick start instructions refer to build_all.sh at this level.

So by moving this into a subdir called Docker-Monolith, we can make it abundantly clear.

Reply:
@hnataraj Usually, Dockerfile is about the build phase and all the build phase files are located at the root of a folder, which the file is building. I think this is the ideal place for the Monolith Dockerfile since this is building all services of crAPI.

```
crAPI/Dockerfile --> for all services in one image
crAPI/services/web --> web service in one image
```

However, we can rename this to `Dockerfile-monolith` or `Dockerfile-crapi-all` if that makes it more clear. I'm fine with either name, it's just the default name is intuitive and keeps different builds steps also easy.
The deploy folder usually is for deployment-related files like shell scripts, compose files and k8s.

Reply:
If you look at current deploy folder, it has a docker sub-dir and all the instructions are aligned with such an arrangement.

By putting a dockerfile at the root you are breaking this arrangement.

thats why I was suggesting to put it under deploy and keep things uniform.

if you feel its ok to break the arrangement, then its fine.

when we add instructions for monolithic build, we will refer to the root folder for this, whereas all other deploys will be in the deploy folder.

btw the other deploy folders also handle building, and not just deployment alone.

Reply:
Fixed. Check the quick start instructions now.